clients/web/src/templates/body/frontPage.pug

```diff
@@ -7,43 +7,75 @@
-        |  link in the upper right corner. If you already have a user, click the
-        a.g-login-link  Log In
-        |  link. Otherwise, you will only be able to see publicly visible data
-        |  in the system.
-    else
-      p.g-frontpage-paragraph
-        |  You are currently logged in as
-        b  #{currentUser.get('login')}
-        | . You can view your
-        a.g-my-folders-link  personal data space
-        |  or go to your
-        a.g-my-account-link  user account page.
-
-    p.g-frontpage-paragraph
-      if versionInfo.git
-        | This instance of Girder was built from Git version&nbsp;
-        a.g-git-sha(
-            target="_blank", href=`https://github.com/girder/girder/tree/${versionInfo.SHA}`)= versionInfo.shortSHA
+
+    ul
+      if !currentUser
+        li You are currently browsing anonymously.
+        ul
+          li
+            | To explore the data hosted on this server, start with the
+            a.g-collections-link  Collections
+            |  page.
+
+          li
+            | To create an account, use the
+            a.g-register-link  Register
+            |  link in the upper right corner.
+
+          li
+            | If you already have an account you can
+            a.g-login-link  log in
+            |  with the link in the upper right corner.
+      else
+        li
+          | You are currently logged in as
+          b  #{currentUser.get('login')}
+          | .
+
+          ul
+            li
+              | To explore the data hosted on this server, start with the
+              a.g-collections-link  Collections
+              |  page.
+
+            li
+              | To explore your data, go to your
+              a.g-my-folders-link  personal data space
+              |  or your
+              a.g-my-account-link  user account page
+              | .
+
+      li
+        | To learn more about Girder, including how you can host your own
+        | instance either locally or in the cloud, see the
+        a(target="_blank", href="http://girder.readthedocs.io/en/latest/user-guide.html")  User Guide
+        | , the
+        a(target="_blank", href="http://girder.readthedocs.io/en/latest")  full documentation
+        | , or visit the
+        a(target="_blank", href="https://github.com/girder/girder")  GitHub repository
+        | .
+
+      li
+        | To use Girder's REST API to interact with this server, check out the
+        a(href=apiRoot)  interactive web API docs
         | .
+
+      li
+        | Girder is a part of
+        a(target="_blank", href="http://resonant.kitware.com")  Resonant
+        | , Kitware's open-source platform for data management, analytics, and
+        | visualization. To learn more, you can read our series of
+        a(target="_blank", href="https://blog.kitware.com/tag/resonant/")  blog posts
+        |  about Resonant, or
+        a(target="_blank", href="http://www.kitware.com/company/contact_kitware.php")  contact us
```

Comment:
Should this link to https://www.kitware.com/contact-us/ instead? (Your link is a 301 for this).